addons/mass_mailing/static/src/js/mass_mailing_snippets.js

```diff
@@ -171,4 +171,19 @@ options.registry.ImageOptimize.include({
     },
 });
 
+options.registry.Parallax = options.Class.extend({
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    async _computeWidgetVisibility(widgetName, params) {
+        // Parallax is not supported in emails.
+        return false;
```

Comment:
Obligatory question: What happens if a customer has the new js but the old xml ? In this case I think it does nothing because this particular widget is not instantiated in this context so this js code is never executed.

Reply:
Then they won't need this code because they don't have the snippet option. This code is to make sure the Parallax snippet option doesn't appear in mass_mailing and without that xml, it doesn't.

Reply:
@dmo-odoo Then they won't need this code because they don't have the snippet option. This code is to make sure the Parallax snippet option doesn't appear in mass_mailing and without that xml, it doesn't.